ios/CozyReactNative/Info.plist

```diff
@@ -62,13 +66,20 @@
 	</dict>
 	<key>NSCameraUsageDescription</key>
 	<string>Afin de prendre des photos pour Drive ou MesPapiers afin d'importer des documents</string>
+	<key>NSFaceIDUsageDescription</key>
+	<string>Permet le déverouillage de l'application en utilisant FaceID</string>
 	<key>NSLocationWhenInUseUsageDescription</key>
-	<string/>
+	<string></string>
 	<key>UIAppFonts</key>
 	<array>
 		<string>Lato-Bold.ttf</string>
 		<string>Lato-Regular.ttf</string>
 	</array>
+	<key>UIBackgroundModes</key>
+	<array>
+		<string>fetch</string>
```

Comment:
Do we really need `fetch`?

Reply:
I blindly followed rnfirebase setup for this https://rnfirebase.io/messaging/usage/ios-setup#enable-background-modes